synapse/lib/aha.py

```diff
@@ -136,6 +137,16 @@ async def getAhaUrls(self, user='root'):
             return ()
```

Comment:
We can drop 
```
if ahaurls is None:
    return ()
```
 because `ahaurls` can never be None, a test I wrote exposed this.

Reply:
Marking this conversation as resolved. The change is in the mirror tool PR.